alvr/client_openxr/src/interaction.rs

```diff
@@ -407,14 +414,13 @@ impl InteractionContext {
 
         // Note: We cannot enable multimodal if fb body tracking is active. It would result in a
         // ERROR_RUNTIME_FAILURE crash.
-        if config.body_tracking.is_none()
-            && config.prefers_multimodal_input
-            && extra_extensions::resume_simultaneous_hands_and_controllers_tracking_meta(
-                &self.xr_session,
-            )
-            .is_ok()
-        {
-            self.multimodal_hands_enabled = true;
+        if config.body_tracking.is_none() && config.prefers_multimodal_input {
+            if let Some((handle, enabled)) = &mut self.multimodal_hands {
+                if !*enabled {
+                    handle.resume().ok();
+                    *enabled = true;
```

Comment:
This right here is the only place where having the bool nested would prevent any sort of programming bugs
and I think
```rust
if let Some(handle) = ... {
handle.resume().ok();
self.mm_enabled = true;
}
```
is easy enough to not mess up